### PR TITLE
yeps

### PR DESCRIPTION
**Pull request by evheniy**
I added tests (with router and without router) for my own framework: YEPS.

### CODE REVIEW

File: benchmarks/yeps-router.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const App = require('yeps')
+const Router = require('yeps-router')
+
+const app = new App()
+const router = new Router()
+
+router.get('/').then(async (ctx) => {
+  ctx.res.statusCode = 200
+  ctx.res.end('{"hello":"world"}')
```
Comment (mcollina) [inline benchmarks/yeps-router.js]: Can you please use `JSON.stringify` here?

Also, you'll need to set the proper header like in https://github.com/fastify/benchmarks/blob/1e6d459dad67ca7f957d086f9cb9830b14533d44/benchmarks/bare.js#L4-L5.

File: benchmarks/yeps.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const http = require('http')
+const App = require('yeps')
+
+const app = new App()
+
+app.then(async ctx => {
+  ctx.res.statusCode = 200
+  ctx.res.end('{"hello":"world"}')
```
Comment (mcollina) [inline benchmarks/yeps.js]: Can you please use `JSON.stringify` here?

Also, you'll need to set the proper header like in https://github.com/fastify/benchmarks/blob/1e6d459dad67ca7f957d086f9cb9830b14533d44/benchmarks/bare.js#L4-L5.

**Comment by mcollina** [comment]
Why did you close this? It seems useful to have this in here.

**Comment by evheniy** [comment]
More than week I wait for merging.
Maybe it's not useful.

**Comment by mcollina** [comment]
It’s open source ;).

**Pull request merged**